### DISCUSSION TITLE
trans/ambient-mesh-can-sidecar-less-istio-make-applications-faster/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/rootsongjc.github.io#356] [Announcements]
# trans/ambient-mesh-can-sidecar-less-istio-make-applications-faster/

探索如何使用 Fortio 与 Istio 集成，在使用 Bookinfo 应用和流行的 DevOps 工具如 Kubernetes、Prometheus 和 Grafana 的微服务架构中进行高效的性能测试和监控。

https://jimmysong.io/trans/ambient-mesh-can-sidecar-less-istio-make-applications-faster/

**Comment by rootsongjc** [comment]
Ambient还要考虑一个问题就是跨区域网络流量费用：Ztunnel 优先在同一可用区 (AZ) 内路由流量，减少跨区费用；waypoint 需跨 AZ 部署，以降低延迟和成本。
